cyphejor.el

```diff
@@ -148,9 +156,15 @@ information."
   (funcall (if cyphejor-mode #'add-hook #'remove-hook)
            'after-change-major-mode-hook
            #'cyphejor--hook)
+  (funcall (if cyphejor-mode #'add-hook #'remove-hook)
```

Comment:
Can you remind me how to reproduce a situation when the current code fails to handle fundamental? I tried to switch to fundamental after creating a buffer with `M-x fundamental-mode` and it worked (name of mode became just "F"). I remember there should be a case when it'll stay with "Fundamental" highlighter, but I don't remember how to reproduce it.

Reply:
It may be better to leave fundamental alone, because it's supposed to be a "vanilla" enviroment and having many hooks just for this one special mode/using hacks is not a good thing IMO.

Reply:
The case when fundamental isn’t handled arises when you do `'M-x switch-to-buffer` and enter a buffer name that doesn’t exist yet. I’m not sure there’s something inherently evil in switching the mode name for fundamental mode buffers. It just seems they are tricky to catch.

Reply:
Strange, for me it creates a buffer in `text-mode`...